### ISSUE TITLE
SSL Certificate (HTTPS Availability)

### ISSUE THREAD

**Issue by TekkitCommando** [DragonetMC/DragonProxy#52]
@DefinitlyEvil I know you wanted ssl certification for dragonet so I thought I would tell you, I think if Dragonet gets a few more donations (We have only had one) I can talk to some of my partners and try to find and setup ssl encryption for the website.

**Comment by mastercoms** [comment]
What about let's encrypt or startssl?

**Comment by TekkitCommando** [comment]
@mastercoms I will check those out.

**Comment by DefinitlyEvil** [comment]
Free SSL veers are not safe and only lasts for one year I remember.

**Comment by mastercoms** [comment]
They are definitely safe and are free forever

**Comment by DefinitlyEvil** [comment]
@mastercoms Okay, could u plz help to get one? I'm too busy these days.

**Comment by TekkitCommando** [comment]
@DefinitlyEvil I will do it, mastercoms is probly busy xD

**Comment by TekkitCommando** [comment]
@DefinitlyEvil I will do it, mastercoms is probly busy xD

**Comment by Plazmaz** [comment]
Letsencrypt is safe, and is an acceptable solution to this.

**Comment by DefinitlyEvil** [comment]
@TekkitCommando How's the progress?

**Comment by TekkitCommando** [comment]
@DefinitlyEvil I am sure I can get it all setup and ready tomorrow

**Comment by DefinitlyEvil** [comment]
@TekkitCommando Okay. ;) Gonna setup CLS now.

**Comment by markstinson** [comment]
https://letsencrypt.org is totally free, founded by Google and other major companies, and will be handed over the EFF.org in a short while. There's no need to wait on getting your publicly trusted SSL Cert for your site or DragonProxy.

To help fund your efforts, you might even consider some sponsorship tiering which could help with development for a Bukkit/Spigot plugin (making it universal & integrated), Android App, etc:
free - at the moment and as always
$0.99 - bronze star donation
$1.99 - silver star donation
... etc,

Maybe donation benefits could include a coupon/discount/free purchase via Google Play for future Android App proxy, Windows 10 App discount, etc.
(OR not since it's the thought that counts being it helps community, app improvement & security fixes.)

**Comment by TekkitCommando** [comment]
@markstinson Yeah LetsEncrypt setup is on my huge list of things to do with Byte Powered and DragonProxy xD

**Comment by TekkitCommando** [comment]
@DefinitlyEvil Oh that reminds me, what do you think about Dragonet teams website using LaraFlex in the future (Like a month or 2) instead of a static html site? Also there are some "regulations" that are being implemented into Byte Powered owned/funded sites/projects that will take 30 seconds to implement which I will tell you on skype about later.

**Comment by DefinitlyEvil** [comment]
@TekkitCommando 
Sure sure, good idea. ;D 
But you have to let me back up the old code and keep the CLS thing. xD

**Comment by TekkitCommando** [comment]
Ok xD

**Comment by TheDiamondYT1** [comment]
\*5 months later*

**Comment by TheDiamondYT1** [comment]
Lets not use LaraFlex. WordPress can do :)

**Comment by TekkitCommando** [comment]
Lol this is old and I have already accepted WordPress xD

**Comment by TheDiamondYT1** [comment]
Add ssl certificate please

**Comment by TekkitCommando** [comment]
I was working on that yesterday. When I have the time I will finish it.

**Comment by TheDiamondYT1** [comment]
Wtf https://dragonet.org/ @TekkitCommanndo

**Comment by TheDiamondYT1** [comment]
@TekkitCommando ^

**Comment by TekkitCommando** [comment]
Did def pay for Dragonet.org?

**Comment by TekkitCommando** [comment]
That is my old hosting service site.

**Comment by TekkitCommando** [comment]
@TheDiamondYT1 Make sure he paid or it could have defaulted to one of my old sites on the machine.

**Comment by DefinitlyEvil** [comment]
Yes have have that domain and renewed.

**Comment by TekkitCommando** [comment]
Alright. Site is working properly now friend.

**Comment by TheDiamondYT1** [comment]
Https shows mc server bank wtf

**Comment by TekkitCommando** [comment]
Probably because it was the only site setup for https back in the day.

**Comment by TheDiamondYT1** [comment]
Can you fix it

**Comment by TekkitCommando** [comment]
No need rn. Just don't use HTTPS :P

**Comment by TheDiamondYT1** [comment]
Is it hard to fix

**Comment by TekkitCommando** [comment]
Not really, just time consuming.

**Comment by Aericio** [comment]
@TekkitCommando Why not use CloudFlare? There are tutorials on how to setup SSL with Page Rules.

**Comment by TheDiamondYT1** [comment]
i already tried getting him tto switch nameservers for cloudflare.

i have an account for dragonet if you want the credentials ethan

**Comment by TekkitCommando** [comment]
@TheDiamondYT1 I already said that is Def's job. He is lead developer of Dragonet.

**Comment by Aericio** [comment]
I can set it up, but I rather tell you how to do it, you probably won't trust me

**Comment by TheDiamondYT1** [comment]
well def is too busy

**Comment by TekkitCommando** [comment]
@Aericio I know how to do it but @TheDiamondYT1 it is still @DefinitlyEvil 's decision to do it. I am a pretty busy person too btw. Also get on Skype and explain to me about that project you want me to develop the user backend for.  @Aericio I don't use 55hl or whatever the crap that is xD They are hosted on personal machines but thanks for the reference to ThinkWithGoogle. Def can use it if he wants.

**Comment by Aericio** [comment]
I didn't ask for a user backend, I am just saying you can set it up easily.

**Comment by TekkitCommando** [comment]
@Aericio Then I probably wasn't talking to you ;) lol

**Comment by DefinitlyEvil** [comment]
@TekkitCommando @TheDiamondYT1 While I am busy, you guys are in charge. ;) Feel free to take actions instead of waiting for me. xD

**Comment by Aericio** [comment]
Sorry :P

**Comment by TheDiamondYT1** [comment]
@TekkitCommando You heard the man. I will skype you the cloudflare details

**Comment by TekkitCommando** [comment]
@TheDiamondYT1 Wait.. we are talking about the website right? If so that already has cloudflare setup. So just message Phill on Skype and ask him about.

**Comment by TheDiamondYT1** [comment]
@TekkitCommando The nameservers are on cloudflare?

**Comment by TekkitCommando** [comment]
@TheDiamondYT1 I am not sure how Phill has got it setup but you can ask him.

**Comment by Aericio** [comment]
Setup SSL:

- Go to Page Rules

- Click on "Create Page Rule"

- Make it to this: (+ Add Setting = Always Use HTTPS, then do what I did)

![image](https://cloud.githubusercontent.com/assets/16523741/17191814/80ddcffe-53e7-11e6-8bab-aefc52cbf287.png)

**Comment by TheDiamondYT1** [comment]
We know
